### PR TITLE
* MOD: promise load - build query

### PR DESCRIPTION
**Pull request by mullerzs**

### CODE REVIEW

File: model.coffee [hunk 1]
```diff
@@ -240,26 +240,30 @@ class AlienModel extends AlienCommander
...
-    for name, value of filters
-      if _.isFunction value
-        value.call @, s, op, filters, db_options, qb, name, value
-      else if _.isArray value
-        qb.where name, 'in', value
-      else if _.isObject value
-        qb.where name, operator, argument for operator, argument of value
-      else
-        qb.where name, value
+  _buildQuery: (s, op, filters, db_options, qb) ->
```
Comment (SaturnV) [inline model.coffee]: Legyen db_op, a maybeRefresh-hez hasonóan

File: model.coffee [hunk 2]
```diff
@@ -240,26 +240,30 @@ class AlienModel extends AlienCommander
...
+        if _.isFunction value
+          value.call @, s, op, filters, db_options, qb, name, value
+        else if _.isArray value
+          qb.where name, 'in', value
+        else if _.isObject value
+          qb.where name, operator, argument for operator, argument of value
+        else
+          qb.where name, value
+    else
+      qb.where 'id', filters
```
Comment (SaturnV) [inline model.coffee]: @idFields[0]

File: model.coffee [hunk 3]
```diff
@@ -268,7 +272,7 @@ class AlienModel extends AlienCommander
     Promise.join p_id, p_db_options,
       (id, db_options) =>
         @bookshelfModel.query (qb) =>
-                         qb.where id
+                         @_buildQuery s, op, id, db_options, qb
                            .limit 2
```
Comment (SaturnV) [inline model.coffee]: Legyen iffel a builderben

File: model.coffee (hunk 1 above)
Comment (mullerzs) [reply model.coffee]: done

File: model.coffee (hunk 2 above)
Comment (mullerzs) [reply model.coffee]: done

File: model.coffee (hunk 3 above)
Comment (mullerzs) [reply model.coffee]: done

**Review by SaturnV — APPROVED** [review]

**Pull request merged**